src/planscape/core/models.py

```diff
@@ -15,8 +17,40 @@ class Meta:
         abstract = True
 
 
+class AliveObjectsManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(deleted_at=None)
+
+
 class DeletedAtMixin(models.Model):
-    deleted_at = models.DateTimeField(null=True)
+    objects = AliveObjectsManager()
+    dead_or_alive = models.Manager()
+
+    deleted_at = models.DateTimeField(
+        verbose_name="Deleted at",
+        help_text="Define if the entity has been deleted or not and when",
+        null=True,
+    )
+
+    def delete(self, *args, **kwargs):
+        """Soft delete an entity."""
```

Comment:
Maybe this comment belongs to  the `else` below not when checking for `hard_delete`